docs/MODULES.md

```diff
@@ -3,6 +3,7 @@
 A short summary of each module is directly below. Documentation for specific abilities of modules, or reference commands are further below.
 
 ## Table of Contents
+
```

Comment:
This will get removed by the tool I use to do it automatically, I would not do this.
```suggestion
```